client/src/containers/CampaignAdmin/tests/CampaignAdmin-test.js

```diff
@@ -40,14 +42,6 @@ function makeProps(obj = {}) {
         view: null
       }
     },
-    sectionConfig: {
```

Comment:
We should not be removing stuff here in a linting PR, please revert

Reply:
I fix it to follow [no-dupe-keys](https://eslint.org/docs/latest/rules/no-dupe-keys) rule.
This object was initially created incorrectly and test shouldn't work properly. Object in JS cannot have duplicate keys, but if it has the last value will be saved. So this object after return looks like
```js
{
...
sectionConfig: {
   form: {
        campaignCreateForm: {
          schemaUrl: '/my/schema'
        }
      },
      reactRoutePath: '/campaigns',
    },
}
...
```
instead of 
```js
  sectionConfig: {
      reactRoutePath: '',
      reactRoutePath: '/campaigns',
      publishEndpoint: {
        url: '/',
        method: 'get',
      },
      form: {
        EditForm: {
          schemaUrl: 'schemaEditForm'
        },
        campaignCreateForm: {
          schemaUrl: '/my/schema'
        }
      }
    },
```
So I just combine all together. 
I don't know what we are going to archive in this test logic, if you think that previous object was created correctly, I'll glad to revert.